modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.ts

```diff
@@ -368,22 +375,24 @@ export default class ScenegraphLayer<DataT = any, ExtraPropsT extends {} = {}> e
       if (node instanceof ModelNode) {
         const {model} = node;
         model.setInstanceCount(numInstances);
+
+        const pbrProjectionProps = {
+          // Needed for PBR (TODO: find better way to get it)
+          // eslint-disable-next-line camelcase
+          camera: model.uniforms.cameraPosition as [number, number, number]
+        };
         const scenegraphProps: ScenegraphProps = {
           sizeScale,
           sizeMinPixels,
           sizeMaxPixels,
           composeModelMatrix: shouldComposeModelMatrix(viewport, coordinateSystem),
           sceneModelMatrix: worldMatrix
         };
-        // TODO replace with shaderInputs.setProps({pbr: u_Camera}) once
-        // luma pbr module ported to UBO
-        model.setUniforms({
-          // Needed for PBR (TODO: find better way to get it)
-          // eslint-disable-next-line camelcase
-          u_Camera: model.uniforms.cameraPosition
-        });
 
-        model.shaderInputs.setProps({scenegraph: scenegraphProps});
+        model.shaderInputs.setProps({
+          pbrProjection: pbrProjectionProps,
+          scenegraph: scenegraphProps
```

Comment:
Audit question: Starting to think that calling the layers uniform definitions `scenegraphLayer` would be clearer here? (for a moment I was wondering if we had introduced a `scenegraph` shader module)